priv/repo/migrations/20170326201602_add-user-group-is-admin.exs

```diff
@@ -0,0 +1,21 @@
+defmodule :"Elixir.Pairmotron.Repo.Migrations.Add-user-group-is-admin" do
+  use Ecto.Migration
+
+  def up do
+    alter table(:users_groups) do
+      add :is_admin, :boolean
+    end
+
+    execute "UPDATE users_groups SET is_admin = FALSE where is_admin IS NULL"
+
+    alter table (:users_groups) do
+      modify :is_admin, :boolean, null: false, default: false
```

Comment:
This is to avoid a full table rewrite when adding the column with a default not-null, right? In this case, since you are adding the column in the same transaction you are setting the value, aren't you also doing a full table rewrite?